python_modules/libraries/dagster-aws/dagster_aws/pipes/message_readers.py

```diff
@@ -357,10 +379,14 @@ def messages_are_readable(self, params: PipesParams) -> bool:
         if self.log_group is not None and self.log_stream is not None:
             # check if the stream actually exists
             try:
-                self.client.describe_log_streams(
+                resp = self.client.describe_log_streams(
                     logGroupName=self.log_group,
                     logStreamNamePrefix=self.log_stream,
                 )
+
+                if not resp.get("logStreams", []):
+                    return False
+
                 return True
```

Comment:
for better readability

```python
else:
    return False
````

Reply:
I think I've implemented this, wasn't sure if I exactly understood correctly though